src/scss/_main.scss

```diff
@@ -864,32 +892,87 @@ nav {
   }
 }
 
-.logs .-focus {
-  background: var(--color-focus);
+.logs-table::-webkit-resizer {
+  background: var(--color-bg-dark);
+
+  border: 2px solid var(--color-bg-dark);
+  border-bottom: 3px solid var(--color-bg-light);
+  border-right: 3px solid var(--color-bg-light);
 }
 
-.line a {
-  text-decoration: none;
+.logs-table::-webkit-scrollbar {
+  width: 0.7rem;
+}
 
-  &:hover,
-  &:focus {
-    text-decoration: underline;
+.logs-table::-webkit-scrollbar-thumb {
+  background-color: var(--color-bg-light);
+  border-radius: 1rem;
+  border: 2px solid var(--color-bg-dark);
+}
+
+.logs-header {
+  padding: 0.25rem 0;
+  border-bottom: 1px solid var(--color-slate);
+  background: var(--color-bg-dark);
+  width: 100%;
+  z-index: 1;
+}
+
+.logs-header .actions {
+  padding: 0.3rem;
+
+  div {
+    justify-content: flex-end;
+  }
+
+  .button {
+    margin-right: 0.3rem;
+  }
+}
+
+.tracker {
+  opacity: 0;
+}
+
+.logs-sidebar {
+  height: 100%;
+  position: absolute;
+  right: 1rem;
+  padding-bottom: 2rem;
+}
+
+.logs-sidebar .inner-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
```

Comment:
i believe these can be safely removed

Reply:
taken care of 👍  also removed some other redundant style
https://github.com/go-vela/ui/pull/197/commits/0ebbf0d05a92a3deeec6ac94e30f4f35398e1ffd